popup.html

```diff
@@ -1,51 +1,75 @@
-<!Doctype html>
+<!DOCTYPE html>
 <html>
 
 <head>
-  <title>Channel-i</title>
-  <meta http-equiv="X-UA-Compatible" content="chrome=1">
-  <meta name="description" content="">
-  <meta charset="utf-8">
-  <link href='css/popup.css' rel='stylesheet' type='text/css'>
-  <style>
-    #user_box, #login, #main {
-      display:none;
-    }
-  </style>
+    <title>Channel-i</title>
+
+    <meta name="description" content="A gateway to Channel-i ever present in your browser">
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!-- Semantic UI CSS -->
+    <link rel="stylesheet" type="text/css" href="http://semantic-ui.com/dist/semantic.min.css">
```

Comment:
I removed the JS associated with that framework, will remove all the unused CSS styles as well, that should reduce the burden